tests/ternaries/indent.js

```diff
@@ -1 +1,15 @@
 aaaaaaaaaaaaaaa ? bbbbbbbbbbbbbbbbbb : ccccccccccccccc ? ddddddddddddddd : eeeeeeeeeeeeeee ? fffffffffffffff : gggggggggggggggg
+
+aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
```

Comment:
You could set `printWidth: 40` instead of very long strings.

Reply:
It'll cause unnecessary diffs, I think it'd be better to not touch unrelated snapshots?

Reply:
Yeah that's true.